test/common_utils.py

```diff
@@ -501,24 +523,25 @@ def make_image_loader(
     constant_alpha=True,
     memory_format=torch.contiguous_format,
 ):
+    if not constant_alpha:
+        raise ValueError("This should never happen")
```

Comment:
Per comment. This will be removed with all the other "old" v2 functionalites.

Reply:
To clarify: we'll remove the entire `make_image_loader` util, right?

Reply:
Yes. Everything postfixed `_loader` as well as all support classes and functions that come with that.